scripts/prettier/prettier-helpers.js

```diff
@@ -46,9 +46,9 @@ function runPrettier(files, config = {}) {
 
   if (runAsync) {
     return exec(cmd, undefined, undefined, process);
-  } else {
-    execSync(cmd);
   }
+
+  return execSync(cmd);
```

Comment:
being explicit also adds better TS inference, now the function returns properly Promise<...> Or void